js/main.js

```diff
@@ -49,19 +45,19 @@ function setBgGreet() {
     greeting.textContent = "Good Evening";
     document.body.style.color = "white";
   }
-}
+};
 
 // Get Name
-function getName() {
+let getName = () => {
   if (localStorage.getItem("name") === null) {
     myName.textContent = "[Enter Name]";
```

Comment:
May be ternary operator can be used here ?

Reply:
Ok. Agree. Changed.

Reply:
Thanks